apps/website/docs/infrastructure/monorepository-setup.md

```diff
@@ -0,0 +1,122 @@
+
+### CODEOWNERS Definition
+
+Define a `CODEOWNERS` file to manage repository ownership. A common setup is:
+
+- Your team has control over the entire repository (`*`).
+- Both your team and cloud engineers have control over the `infra` and `.github`
+  paths.
+
+Example `CODEOWNERS` file:
+
+```md
+# See https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file
+
+* <your-team>
+/infra/ <your-team> @engineering-team-cloud-eng
+.github/ <your-team> @engineering-team-cloud-eng
+```
+
+### Creation of Dotfiles
+
+Create the following dotfiles at the root of your repository:
+
+- `.terraform-version`: Specify the Terraform version to use (typically
+  [the latest available](https://developer.hashicorp.com/terraform/install?product_intent=terraform)).
+- `.gitignore`: Add rules to manage Terraform files
+  ([example](https://github.com/pagopa/dx-typescript/blob/main/.gitignore#L1)).
+- `.pre-commit-config.yaml`: Define pre-commit hooks
+  ([example](https://github.com/pagopa/dx/blob/main/.pre-commit-config.yaml)).
+- `.editorconfig`: Enforce consistent coding styles
+  ([example](https://github.com/pagopa/dx-typescript/blob/main/.editorconfig)).
+- `.tflint.hcl`: Configure TFLint for Terraform linting
+  ([example](https://github.com/pagopa/dx/blob/main/.tflint.hcl)).
+- `.trivyignore`: Define rules for Trivy vulnerability scanning
+  ([example](https://github.com/pagopa/dx/blob/main/.trivyignore)).
+
+### GitHub PAT
+
+To apply changes to your repository via Terraform, you need a Personal Access
+Token (PAT) for authentication. A single PAT with the following permissions is
+required for all repositories managed through Terraform:
+
+- `read`: `metadata`
+- `read+write`: `variables`, `administration`, `environments`, `secrets`
+
+If you already have a PAT configured with these permissions, you may skip the
+next section.
+
+#### Creation of GitHub PAT
+
+If you do not already have a Personal Access Token (PAT), follow these steps:
+
+1. Go to your GitHub settings, under `Developer settings`, and create a new
+   fine-grained PAT:
+   - Add these permissions:
+     - `read`: `metadata`
+     - `read+write`: `variables`, `administration`, `environments`, `secrets`
+   - Select `Only select repositories` and add the new repository.
+   - Add a meaningful description like "PAT to manage GitHub locally via
+     Terraform."
+2. Set in your local environment the variable `GITHUB_TOKEN` with the generated PAT value in your CLI
+   profile.
+
+#### Update of GitHub PAT
+
+If you already have the PAT in both your GitHub account and your CLI profile,
+ensure that the new repository is accessible from that PAT.
+
+1. Go to your GitHub settings, under `Developer settings`, and select the
+   existing fine-grained PAT.
+2. Under `Only select repositories`, add your new repository.
+
+:::warning
+
+PATs have an expiration date. [Be sure to renew them periodically](https://pagopa.atlassian.net/wiki/search?text=github%20bot%20pat ).
```

Comment:
here it's stated that they shouldn't: https://pagopa.atlassian.net/wiki/search?text=Github%20bot%20setup%20%2B%20invitation

Reply:
Here's about personal accounts, not bots